crates/ethereum/consensus/src/lib.rs

```diff
@@ -38,12 +38,14 @@ pub use validation::validate_block_post_execution;
 pub struct EthBeaconConsensus<ChainSpec> {
     /// Configuration
     chain_spec: Arc<ChainSpec>,
+    /// Maximum allowed extra data size in bytes
+    pub max_extra_data_size: usize,
```

Comment:
this should be private and instead have a getter and a `fn with_max_extra_data_size` builder style fn